pkg/cli/editor_input_part2.go

```
@@ -0,0 +1,50 @@
+func runEditorInput(inputCfg freeTextInputConfig, fieldName, initialContent string) (string, bool, error) {
+	editorCommand, err := inputCfg.resolveEditorCommand()
+	if err != nil {
+		return "", false, err
+	}
+
+	tempFile, err := os.CreateTemp("", "kit-input-*.md")
+	if err != nil {
+		return "", false, fmt.Errorf("failed to create temp file for %s: %w", fieldName, err)
+	}
+	tempPath := tempFile.Name()
+	defer os.Remove(tempPath)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)